### DISCUSSION TITLE
Skeleton based multi-person behavior detection

### DISCUSSION THREAD

**Discussion by nlxxWangYH** [open-mmlab/mmaction2#2427] [Q&A]
I want to perform spatiotemporal behavior detection (multiple behaviors) on multiple people in the video at the same time, I check the data preparation chapter of the official document, where the dataset for spatiotemporal behavior detection is the AVA dataset, but the dataset does not use skeleton information. In addition, I also noticed that there is a dataset processing part dedicated to behavior recognition based on the skeleton in the official document, and after I checked it, I found that the number of people in its use scenario did not exceed 2, and it was mainly used for behavior recognition and not spatiotemporal behavior detection.
How do I structure my dataset if I want to detect spatiotemporal behavior for multiple people in a video based on skeleton information?
Thank you very much for checking out my problem in your busy schedule, and I hope you can help me solve this simple problem.

**Comment by cir7** [comment]
Currently, MMAction2 doesn't support training skeleton-based spatio-temporal aciton detection models. We provide a simple inference [demo](https://github.com/open-mmlab/mmaction2/blob/main/demo/demo_skeleton.py), maybe you could refer to it

**Reply by nlxxWangYH** [reply]
Thank you for your answer. 
Is there any future plan for mmaction2 to support spatiotemporal behavior detection methods based on skeleton.

**Comment by cir7** [comment]
yes, we have a plan to support it, but can't guarantee a specific time.

**Reply by nlxxWangYH** [reply]
Absolutely good news, thank you for your contributions and look forward to good news in the future.

**Comment by SCsleepy** [comment]
@nlxxWangYH
May I ask if you have implemented it? Can you teach me. Thank you very much.
